website/frontend/apis/index.js

```diff
@@ -1,37 +1,45 @@
 import axios from 'axios';
 import {
-  AIRQLOUD_SUMMARY, NEWSLETTER_SUBSCRIPTION, INQUIRY_URL, EXPLORE_DATA_URL, CAREERS_URL, DEPARTMENTS_URL, TEAMS_URL
+    AIRQLOUD_SUMMARY,
+    NEWSLETTER_SUBSCRIPTION,
+    INQUIRY_URL,
+    EXPLORE_DATA_URL,
+    CAREERS_URL,
+    DEPARTMENTS_URL,
+    TEAMS_URL,
 } from 'config/urls';
 import { HIGHLIGHTS_URL, TAGS_URL } from '../config/urls';
 
-export const getAirQloudSummaryApi = async () => await axios
-  .get(AIRQLOUD_SUMMARY)
-  .then((response) => response.data);
+const headers = {
+    Authorization: process.env.REACT_APP_AUTHORIZATION_TOKEN,
+};
 
-export const newsletterSubscriptionApi = async (data) => await axios
-  .post(NEWSLETTER_SUBSCRIPTION, data)
-  .then((response) => response.data);
+export const getAirQloudSummaryApi = async () =>
+    await axios.get(AIRQLOUD_SUMMARY, { headers }).then((response) => response.data);
```

Comment:
Thanks @Codebmk 
However, this will fail given that the Authorization is not passed in the headers object but rather as a plain variable